packages/wmr/src/cli.js

```diff
@@ -45,21 +52,25 @@ prog
 	.action(opts => {
 		opts.optimize = !/false|0/.test(opts.compress);
 		opts.compress = bool(opts.compress);
-		if (/true/.test(process.env.PROFILE)) opts.profile = true;
+		if (/true/.test(process.env.PROFILE || '')) opts.profile = true;
 		run(start(opts));
 	});
 
 prog.parse(process.argv);
 
 function run(p) {
-	p.catch(err => {
-		const text = err.message || err + '';
-		const stack = errorstacks
-			.parseStackTrace(err.stack)
-			.map(frame => frame.raw)
-			.join('\n');
+	p.catch(catchException);
+}
 
-		process.stderr.write(`\n${kl.red(text)}\n${stack ? kl.dim(stack + '\n\n') : ''}`);
-		process.exit(p.code || 1);
-	});
+function catchException(err) {
+	const text = (err.stack && err.stack.split('\n')[0]) || err.message || err + '';
```

Comment:
The first line of the stack had a nicer error than the actual message with the error I ran into

```
stack[0] = "Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client"
message  = "Cannot set headers after they are sent to the client"
```